src/quo2/components/settings/settings_list/view.cljs

```diff
@@ -0,0 +1,119 @@
+                       colors/neutral-40)}]])
+
+(def chevron-icon
+  [rn/view
+   [quo2.icons/icon :chevron-right
+    {:color (colors/theme-colors
+             colors/neutral-50
+             colors/neutral-40)}]])
+
+(defn switch-button
+  [{:keys [:value
+           on-value-change]}]
+  (js/console.warn on-value-change)
+  [rn/switch
+   {:value                value
+    :on-value-change      (fn [new-value] (on-value-change new-value))
+    ;; TODO: For some reason this prop is not working on ios simulator, spike a little bit more
+    :ios-background-color (colors/theme-colors
+                           colors/neutral-30
+                           colors/neutral-80)
+    ;; TODO: Ask for colors on the track, there is no definition on the design
+    ;; Colors for false value only work on Android
+    ;; :track-color          {:false (colors/theme-colors
+    ;;                               colors/danger
+    ;;                               colors/danger)
+    ;;                       :true  (colors/theme-colors
+    ;;                               colors/danger
+    ;;                               colors/danger)}
+    :style                style/switch-button}]
+)
+
+(def badge-icon
+  [rn/view
+   {:style style/dot}])
+
+(defn right-button
+  [{:keys [title
+           on-press]}]
+  [rn/touchable-opacity
+   {:on-press on-press
+    :style    style/right-button}
+   [text/text
+    {:accessibility-label :settings-right-button-title
+     :number-of-lines     1
+     :weight              :medium
+     :style               style/right-button-title
+     :size                :paragraph-2}
+    title]]
+)
+
+(defn communities-icons
+  [{:keys [data
+           icon-style]}]
+  (let [communities-count (- (count data) 1)]
+    [rn/view
+     {:style {:flex 1 :margin-right 12}}
+     (map-indexed
+      (fn [index {:keys [source accesibility-label]}]
+        [rn/image
+         {:key                source
+          :source             {:uri source}
+          :accesibility-label accesibility-label
+          :style              (merge (style/community-icon (- communities-count index)) icon-style)}])
+      data)]))
+
+(defn settings-list
+  [{:keys [title
+           on-press
+           accessibility-label
+           left-icon
+           chevron
+           switch-props
+           badge
+           button-props
+           communities-props]}]
+  [rn/touchable-without-feedback
+   {:on-press            on-press
+    :accessibility-label accessibility-label}
+   [rn/view
+    {:style style/item-container}
+    (case left-icon
+      ;; TODO: Add Icon Avatar on next variants development
+      :browser-context browser-context-icon
+      nil)
+    [settings-title title]
+    (when switch-props
+      (switch-button switch-props))
+    (when badge badge-icon)
+    (when button-props
+      (right-button button-props))
+    (when communities-props (communities-icons communities-props))
+    (when chevron chevron-icon)]])
```

Comment:
It is common practice to suffix boolean variable names with `?`. You can use `chevron?` instead of `chevron`

https://github.com/status-im/status-mobile/blob/develop/doc/new-guidelines.md#dont-prepend-booleans-with-is-

Reply:
Thanks!